src/main/java/org/springframework/samples/petclinic/ui/MainLayout.java

```diff
@@ -62,16 +61,11 @@ private Tabs createMenu() {
 	}
 
 	private Component[] createMenuItems() {
-		Tab error = new Tab(getTranslation("error"));
```

Comment:
This is some tests I was doing with error implementation that ended accidentally in previous PR, removing it now